build/tools/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -19,6 +19,59 @@ public static void Run(params string[] outputDirectories)
             var assemblies = new List<Assembly>();
             assemblies.Add(typeof(Instrumentation).Assembly);
 
+            // find all methods in Datadog.Trace.ClrProfiler.Managed.dll with [InstrumentMethod]
+            // and create objects that will generate correct JSON schema
+            var callTargetIntegrations = from assembly in assemblies
+                                         from wrapperType in assembly.GetTypes()
+                                         let attributes = wrapperType.GetCustomAttributes<InstrumentMethodAttribute>(inherit: false)
+                                         where attributes.Any()
+                                         from attribute in attributes
+                                         let integrationName = GetIntegrationName(wrapperType)
+                                         orderby integrationName
+                                         group new
+                                         {
+                                             assembly,
+                                             wrapperType,
+                                             attribute
+                                         }
+                                             by integrationName into g
+                                         select new
+                                         {
+                                             name = g.Key,
+                                             method_replacements = from item in g
+                                                                   select new
+                                                                   {
+                                                                       caller = new
```

Comment:
Just curious, do we need the caller key in the object after all or can you remove it here without causing the integration parsing to fail?

Reply:
I didn't try to change the schema, but is something I hope it will change once we get all the integrations in the new format.